packages/select/src/components/FzSelectLabel.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <button
+    disabled
+    test-id="fzselect-label"
+    :title="option.label"
+    :class="computedClass"
+  >
+    {{ option.label }}
+  </button>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import { FzSelectLabelProps } from "../types";
+
+const props = defineProps<{
+  option: FzSelectLabelProps;
+  size: "sm" | "md" | "lg";
+}>();
+
+const staticClass =
+  "text-grey-400 flex items-center text-ellipsis whitespace-nowrap";
```

Comment:
is it useful to have this declared on its own?

Reply:
is cleaner and easier to move into a composable in the future (it share the same style as the action list label)